src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,290 @@
+"""A Gaussian distribution with tanh transformation."""
+from collections import namedtuple
+
+import torch
+from torch.distributions import Normal
+from torch.distributions.independent import Independent
+
+
+class TanhNormal(torch.distributions.Distribution):
+    """A gaussian distribution that has a tanh transformation applied to it.
```

Comment:
Gaussian (caps)

A distribution induced by applying a tanh transformation to a Gaussian random variable.

Reply:
resolved.